salt/fileclient.py

```diff
@@ -672,6 +681,8 @@ def swift_opt(key, default):
             write_body = [None, False, None]
 
             def on_header(hdr):
+                nonlocal dest_etag
+                nonlocal use_etag
```

Comment:
**suggestion** if `use_etag` and `dest_etag` aren't being *set* in this block, Python's LEGB scoping rules means they shouldn't need to be declared `nonlocal`.

Reply:
I usually default to being explicit. Removed.